projects/experimental/components/textfield/textfield.template.html

```diff
@@ -0,0 +1,34 @@
+<ng-container *ngIf="directive?.change$ | async"></ng-container>
+<ng-content select="input"></ng-content>
+<ng-content select="select"></ng-content>
+<ng-content select="label"></ng-content>
+<span
+    class="t-content"
+    (waResizeObserver)="side = $event[0].contentRect.width"
+>
+    <ng-content></ng-content>
+    <button
+        *ngIf="options.cleaner"
+        appearance="icon"
+        iconLeft="tuiIconClose"
```

Comment:
we need to somehow override icon in proprietary theme

Reply:
Sure, I'll add this later on.